docs/sources/alerting/fundamentals/evaluate-grafana-alerts.md

```diff
@@ -108,8 +108,8 @@ When this query is used as the **condition** in an alert rule, then the non-zero
 | {Host=web3,disk=/var} | Normal   |
 
 {{% docs/reference %}}
-[metadata]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/developers/plugins/metadata"
-[metadata]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/developers/plugins/metadata"
+[metadata]: "/docs/grafana/ -> /developers/plugin-tools/reference-plugin-json"
```

Comment:
The `docs/reference` shortcode calls the `relref` function in its implementation. Each of these entries will cause a build error on the website and locally because the `/developers/` content is not part of the build.

I can make the link checking configurable but it might block this PR. Instead I think it might be quicker if you replace each of these links with inline links as they are through the rest of this PR. We don't really need the contextual resolution since the destination is always the same.

Reply:
Fixed.